cg/models/file_data.py

```diff
@@ -0,0 +1,34 @@
+"""Class to hold information about file objects"""
+
+import gzip
+from pathlib import Path
+from typing import Iterable
+
+GZIP_MAGIC_NUMBER = "1f8b"
+
+
+class FileData:
+    """Helper functions for working with files"""
+
+    @staticmethod
+    def is_gzipped(path: Path) -> bool:
+        """Check if file is zipped"""
+        handle = open(path, "rb")
+        if handle.read(2).hex() == GZIP_MAGIC_NUMBER:
+            return True
+        return False
+
+    @staticmethod
+    def is_empty(path: Path) -> bool:
+        """Check if file is empty"""
+        handle = open(path, "rb")
+        if handle.read(1):
```

Comment:
What does "1" stand for? Could you give it a named constant to make it easier to understand?

Reply:
It is just reading the first byte of the file. I can make a comment in the code